Examples/UIExplorer/NavigationExperimental/NavigationAnimatedExample.js

```diff
@@ -74,11 +69,13 @@ class NavigationAnimatedExample extends React.Component {
       <NavigationAnimatedView
         navigationState={navigationState}
         style={styles.animatedView}
-        renderOverlay={(props) => (
+        renderOverlay={(scenes, index, position, layout) => (
```

Comment:
I'm not sure whether you have updated this to the master or not.

If so, could you please keep the API as what it is in master?

```
renderOverlay={(props: NavigationSceneRendererProps) => ?ReactElement {.....}
```

We'd like to unify the props that is used to render sub-component of the navigation otherwise the code will look like this soon:

```
<MyNavigationView
  navigationState={navigationState}
  style={styles.animatedView}
  renderHeader={(scenes, index, position, layout) => ....}
  renderFooter={(index,  scenes, layout) => ....}
  renderSideMenu={(index, layout) => ....}
/>
```

The `props` should be treated as read-only object that can passed around the sub-components (see https://github.com/facebook/react-native/commit/71e59761c950b5b012c1860501f68a96f38a4168)